src/pages/idea-market/IdeaMarketRegister.tsx

```diff
@@ -189,6 +439,46 @@ export const IdeaMarketRegister: React.FC<IdeaMarketRegisterProps> = () => {
                   onClick={() => setCategory('문서 · 글쓰기')}>
                   문서 · 글쓰기
                 </div>
+                <div
+                  className={styles.dropdownItem}
+                  onClick={() => setCategory('미디어 · 콘텐츠')}>
+                  미디어 · 콘텐츠
+                </div>
+                <div
+                  className={styles.dropdownItem}
+                  onClick={() => setCategory('변역 및 통역')}>
+                  번역 및 통역
+                </div>
+                <div
+                  className={styles.dropdownItem}
+                  onClick={() => setCategory('세무 · 법무 · 노무')}>
+                  세무 · 법무 · 노무
+                </div>
+                <div
+                  className={styles.dropdownItem}
+                  onClick={() => setCategory('주문제작')}>
+                  주문제작
+                </div>
+                <div
+                  className={styles.dropdownItem}
+                  onClick={() => setCategory('창업 · 사업')}>
+                  창업 · 사업
+                </div>
+                <div
+                  className={styles.dropdownItem}
+                  onClick={() => setCategory('푸드 및 음료')}>
+                  푸드 및 음료
+                </div>
+                <div
+                  className={styles.dropdownItem}
+                  onClick={() => setCategory('IT · 테크')}>
+                  IT · 테크
+                </div>
+                <div
+                  className={styles.dropdownItem}
+                  onClick={() => setCategory('기타')}>
+                  기타
```

Comment:
배열 만드셔서 map 돌리면 코드가 간결해져요

```suggestion
const OPTIONS = [ '미디어 콘텐츠', '기타', '푸드 및 음료' ... ]

{OPTIOMS.map((option)) => (
         <div
              className={styles.dropdownItem}
              onClick={() => setCategory(option)}>
              {option}
          </div>
  ))}
```